### PR TITLE
Rectify partial match warnings

### PR DESCRIPTION
**Pull request by EmilRehnberg**
Addresses #841 

There's still partial match warnings that pop up but that's from dependency packages.

I went through the tests to identify partial match issues, so if there's some functions that aren't covered, I most likely missed them.

Also, there's one test case `expect_silent` that I had to remove since `bizdays` gives plenty of partial match warnings. 

Please let me know if you want me to squash commits.

### CODE REVIEW

File: R/errors.R [hunk 1]
```diff
@@ -284,8 +284,8 @@ accuracy.default <- function(object, x, test=NULL, d=NULL, D=NULL, f = NULL, ...
         D <- as.numeric(frequency(object$mean) > 1)
       }
       else {
-        d <- as.numeric(frequency(object$x) == 1)
-        D <- as.numeric(frequency(object$x) > 1)
+        d <- as.numeric(frequency(object$xlevels) == 1)
```
Comment (mitchelloharawild) [inline R/errors.R]: I think object$x is appropriate here.
Perhaps use object[["x"]] instead.

Comment (EmilRehnberg) [reply R/errors.R]: Hmm, this did raise partial match warnings, but the `[[` accessor also works!

Comment (mitchelloharawild) [reply R/errors.R]: `$xlevels` is an element of the `tslm()` output, but (most) models will store the original series as `$x`. This code is looking for the data's frequency, and so `x` is the element we want.

**Comment by EmilRehnberg** [comment]
@mitchelloharawild 
I edited the before mentioned `errors` commit.

I ran the tests one by one to identify the partial matches. Hence there might be some uncovered code that I haven't checked. No special convenient software unfortunately.

**Pull request merged**